src/pages/EntityDetails/Model/Model.js

```diff
@@ -139,6 +140,147 @@ const Model = () => {
   };
 
   const LocalAppChips = renderCounts("localApps", modelStatusData);
+  const offersChips = renderCounts("offers", modelStatusData);
+  const remoteAppChips = renderCounts("remoteApps", modelStatusData);
+
+  const localAppTableLength = localApplicationTableRows?.length;
+  const appOffersTableLength = appOffersRows?.length;
+  const remoteAppsTableLength = remoteApplicationTableRows?.length;
+
+  const OffersHeader = () => (
+    <>
+      <span>
+        {appOffersTableLength} {pluralize(appOffersTableLength, "Offer")}
+      </span>
+      <ChipGroup chips={offersChips} descriptor={null} />
+    </>
+  );
+
+  const LocalAppsHeader = () => (
+    <>
+      <span>
+        {localAppTableLength}{" "}
+        {pluralize(localAppTableLength, "Local application")}
+      </span>
+      <ChipGroup chips={LocalAppChips} descriptor={null} />
+    </>
+  );
+
+  const RemoteAppsHeader = () => (
+    <>
+      <span>
+        {remoteAppsTableLength}{" "}
+        {pluralize(remoteAppsTableLength, "Remote application")}
+      </span>
+      <ChipGroup chips={remoteAppChips} descriptor={null} />
+    </>
+  );
+
+  const LocalAppsTable = () => (
+    <>
+      {localAppTableLength > 0 && (
+        <MainTable
+          headers={localApplicationTableHeaders}
+          rows={localApplicationTableRows}
+          className="entity-details__apps p-main-table"
+          sortable
+          emptyStateMsg={"There are no local applications in this model"}
+        />
+      )}
+    </>
+  );
+
+  const AppOffersTable = () => (
+    <>
+      {appOffersTableLength > 0 && (
+        <>
+          <MainTable
+            headers={appsOffersTableHeaders}
+            rows={appOffersRows}
+            className="entity-details__offers p-main-table"
+            sortable
+            emptyStateMsg={"There are no offers associated with this model"}
+          />
+        </>
+      )}
+    </>
+  );
+
+  const RemoteAppsTable = () => (
+    <>
+      {remoteAppsTableLength > 0 && (
+        <>
+          <ChipGroup chips={remoteAppChips} descriptor={null} />
+          <MainTable
+            headers={remoteApplicationTableHeaders}
+            rows={remoteApplicationTableRows}
+            className="entity-details__remote-apps p-main-table"
+            sortable
+            emptyStateMsg={"There are no remote applications in this model"}
+          />
+        </>
+      )}
+    </>
+  );
+
+  const expandedKey = () => {
+    if (appOffersTableLength) {
+      return "offers";
+    }
+    if (localAppTableLength) {
+      return "local-apps";
+    }
+    if (remoteAppsTableLength) {
+      return "remote-apps";
+    }
+  };
+
+  const getContentReveals = () => {
+    return (
+      <>
+        {appOffersTableLength > 0 && (
```

Comment:
Can you add some spacing between the tables to match the design?

![Screen Shot 2021-03-11 at 3 54 40 PM](https://user-images.githubusercontent.com/532033/110860283-3024a780-8282-11eb-8c61-0a3eac73409f.png)

vs https://app.zeplin.io/project/5fd0f96d9233d05449fd5b4c/screen/6038e606463cb3266af5ec9e